qa-niaid.planx-pla.net/manifest.json

```diff
@@ -45,7 +45,7 @@
     "environment": "qaplanetv1",
     "hostname": "qa-niaid.planx-pla.net",
     "revproxy_arn": "arn:aws:acm:us-east-1:707767160287:certificate/c676c81c-9546-4e9a-9a72-725dd3912bc8",
-    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/niaid-clinical-trials-dictionary/0.5.3/schema.json",
+    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/niaid-clinical-trials-dictionary/0.5.4/schema.json",
```

Comment:
next time, fill out the PR template in your [PR](https://github.com/uc-cdis/niaid-clinical-trials-dictionary/pull/25) description, so that the [release](https://github.com/uc-cdis/niaid-clinical-trials-dictionary/releases/tag/0.5.4) description includes release notes 🙂